Enterprise-CMCS/macpro-mdct-mcr
# MLR Add Program Reporting Info (Modal and Backend Functionality)

## Summary

### Description

Adding a new form under `/mlr/program-information/program-entity-listing` that pops up when you click on the **Add program reporting information** button.

### Related ticket

MDCT-2233

### How to test

- Log in as a state user
- Go to `MLR`
- Create a new report
- Navigate to `/mlr/program-information/program-entity-listing`
- Click on **Add new program reporting information**
- Fill in the form!

With this update, the form is being stored in the DB / S3 buckets, but it won't be displayed just yet (stay tuned), although the **Total MLR reports** counter should go up!

### Important updates

N/A

### Author checklist
